commons/src/main/java/org/infinispan/commons/util/Util.java

```diff
@@ -1108,8 +1110,11 @@ public static void renameTempFile(File tempFile, File lockFile, File dstFile, Bi
       FileLock lock = null;
       try (FileOutputStream lockFileOS = new FileOutputStream(lockFile)) {
          lock = lockFileOS.getChannel().lock();
-         if (!tempFile.renameTo(dstFile)) {
-            renameFailed.accept(tempFile, dstFile);
+         try {
+             Files.move(tempFile.toPath(), dstFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
+         }
+         catch (Exception ex) {
```

Comment:
the `catch` should be in the same line as` } `
also, it would be nice to log the exception as well :)

Reply:
It seems that the renameFailed biconsumer is handling the error and already logs everything. Should we also log the exception?

Reply:
yes.